packages/ruby/spec/readme/metrics_spec.rb

```diff
@@ -318,129 +316,129 @@ def app
-      options = {api_key: "key", buffer_length: "1"}
+    it 'raises when buffer_length is not an integer' do
+      options = { api_key: 'key', buffer_length: '1' }
       expect {
-        Readme::Metrics.new(noop_app, options)
+        described_class.new(noop_app, options)
       }.to raise_error(
         Readme::Errors::ConfigurationError,
         Readme::Errors::BUFFER_LENGTH_ERROR
       )
     end
 
-    it "raises when development is not a boolean" do
-      options = {api_key: "key", development: "true"}
+    it 'raises when development is not a boolean' do
+      options = { api_key: 'key', development: 'true' }
       expect {
-        Readme::Metrics.new(noop_app, options)
+        described_class.new(noop_app, options)
       }.to raise_error(
         Readme::Errors::ConfigurationError,
         Readme::Errors::DEVELOPMENT_ERROR
       )
     end
 
-    it "raises when the logger does not respond to the correct messages" do
-      options = {api_key: "key", logger: Class.new}
+    it 'raises when the logger does not respond to the correct messages' do
+      options = { api_key: 'key', logger: Class.new }
       expect {
-        Readme::Metrics.new(noop_app, options)
+        described_class.new(noop_app, options)
       }.to raise_error(
         Readme::Errors::ConfigurationError,
         Readme::Errors::LOGGER_ERROR
       )
     end
   end
 
-  describe "group validation" do
+  describe 'group validation' do
     before do
       stub_request(:post, Readme::Metrics::ENDPOINT)
       allow(Thread).to receive(:new).and_yield
     end
 
-    it "supports the api_key field in lieu of the deprecated id field" do
+    it 'supports the api_key field in lieu of the deprecated id field' do
       def app
-        json_app_with_middleware({}, {id: :empty, api_key: "8675309"})
+        json_app_with_middleware({}, { id: :empty, api_key: '8675309' })
       end
 
-      post "/api/foo"
+      post '/api/foo'
 
-      expect(WebMock).to have_requested(:post, Readme::Metrics::ENDPOINT)
-        .with { |request| validate_json("readmeMetrics", request.body) }
+      expect(WebMock).to(have_requested(:post, Readme::Metrics::ENDPOINT)
+        .with { |request| validate_json('readmeMetrics', request.body) })
     end
 
-    it "throws when provided an unsupported field" do
+    it 'throws when provided an unsupported field' do
       def app
-        json_app_with_middleware({}, {nananana: "booboo"})
+        json_app_with_middleware({}, { nananana: 'booboo' })
       end
 
-      post "/api/foo"
+      post '/api/foo'
 
       expect {
-        WebMock.to have_requested(:post, Readme::Metrics::ENDPOINT)
-          .with { |request| validate_json("readmeMetrics", request.body) }
-      }.to raise_error
+        WebMock.to(have_requested(:post, Readme::Metrics::ENDPOINT)
+          .with { |request| validate_json('readmeMetrics', request.body) })
+      }.to raise_error # rubocop:disable RSpec/UnspecifiedException
```

Comment:
This test could pass if you had a syntax error someone within in the expect block. It's probably better to be more explicit about the error.

Reply:
This test doesn't actually work at the moment because `WebMock.to` isn't a valid method. I'm going to remove the test because it doesn't seem like we actually throw exceptions for unsupported/unknown config fields being supplied -- nor do I think we really should bother with that.

Reply:
Oh maybe this was a typo or bad copy/paste?